qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/presenters/SearchCustomerPresenter.groovy

```diff
@@ -22,9 +22,9 @@ class SearchCustomerPresenter {
         this.searchCustomerViewModel = searchCustomerViewModel
         //ToDo Remove this list as soon as the backend is implemented
         List<Customer> customerList = new ArrayList<Customer>()
-        def customer1 = new Customer("Ash", "Ketchum", AcademicTitle.NONE, "gottacatchem@all.de", [])
-        def customer2 = new Customer("Samuel", "Oak", AcademicTitle.PROFESSOR, "giveMeMyPokemon@geezer.de", [])
-        def customer3 = new Customer("Pikachu", "Pichu", AcademicTitle.DOCTOR, "Pokdex@25.de", [])
+        def customer1 = new Customer.Builder("Ash", "Ketchum", "gottacatchem@all.de").title(AcademicTitle.NONE).build()
+        def customer2 = new Customer.Builder("Samuel", "Oak", "giveMeMyPokemon@geezer.de").title(AcademicTitle.PROFESSOR).build()
+        def customer3 = new Customer.Builder("Pikachu", "Pichu", "Pokdex@25.de").title(AcademicTitle.DOCTOR).build()
```

Comment:
Since search customer should already be implemented I would suggest removing the stubbed data here instead of adapting them.

Reply:
Good Point, I would rather do that in a seperate PR though to keep it seperate.